Readme.md

```diff
@@ -1,84 +1,76 @@
-    This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences.
-    Copyright (C) 2012 - 2015 by the BurnMan team
-    This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
-
-    This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
-    You should have received a copy of the GNU General Public License along
-    with this program; if not, write to the Free Software Foundation, Inc.,
-    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+# BurnMan - a thermoelastic and thermodynamic toolkit for Earth and planetary sciences
+<img src="sphinx/burnjack.png", width="256">
+## About
 
-Contact: Ian Rose 
-ian.rose@berkeley.edu
-
-*** About
-
-BurnMan is a lower mantle shear velocity generator constrained by mineral physics.
+BurnMan is a Python library for generating thermodynamic and thermoelastic models of planetary interiors.
 
-Work started by a work-group of the CIDER 2012 workshop in Santa Barbara.
+It began as a working-group at the 2012 CIDER workshop in Santa Barbara.
 
 BurnMan is released under the GNU GPL v2 or newer
 
 Homepage: http://burnman.org
+
 Documentation: http://burnman.org/current-doc
+
 Source code: https://github.com/geodynamics/burnman
 
 Authors (as of 2015, listed alphabetically by first name):
-	Bob Myhill
-	Cayman Unterborn
-	Ian Rose
-	Sanne Cottaar
-	Timo Heister
+* Bob Myhill
+* Cayman Unterborn
+* Ian Rose
+* Sanne Cottaar
+* Timo Heister
 
-*** Requirements
+Contact: Ian Rose 
+ian.rose@berkeley.edu
 
-- Python 2.7 (not Python 3.x)
-- Python modules:
-  NumPy, SciPy, matplotlib
+## Requirements
 
+* Python 2.7 (not Python 3.x)
```

Comment:
Can this be changed to "Python 2.7 and later"? Are there any major problems with 3.x?